flink-table-store-core/src/main/java/org/apache/flink/table/store/file/data/DataFileMeta.java

```diff
@@ -49,6 +57,26 @@ public class DataFileMeta {
     private final long maxSequenceNumber;
     private final int level;
 
+    public DataFileMeta(
```

Comment:
Can we have a static method , maybe just `DataFileMeta.fromAppend`

Reply:
Okay, that seems to be more clear.